django/contrib/admin/widgets.py

```diff
@@ -393,10 +393,12 @@ def __init__(self, rel, admin_site, attrs=None, choices=(), using=None):
         self.db = using
         self.choices = choices
         self.attrs = {} if attrs is None else attrs.copy()
+        self.to_field = self.rel.field.to_fields[0] if hasattr(self.rel.field, 'to_fields') else None
 
     def get_url(self):
         model = self.rel.model
-        return reverse(self.url_name % (self.admin_site.name, model._meta.app_label, model._meta.model_name))
+        url = reverse(self.url_name % (self.admin_site.name, model._meta.app_label, model._meta.model_name))
+        return url if not self.to_field else url + '?to_field=' + self.to_field
```

Comment:
You shouldn't add the attribute to the URL, it should be passed to the DOM via a `data`-attribute and added to the query function the javascript code, as it is done with all other get attributes.
You are "lucky" that this works, but it could break in the future.
I'd also encourage you to always use `urllib.parse.urlencode`.